lib/RemoteAST/RemoteAST.cpp

```diff
@@ -1162,6 +1165,35 @@ class RemoteASTContextConcreteImpl final : public RemoteASTContextImpl {
     if (result) return RemoteAddress(*result);
     return getFailure<RemoteAddress>();
   }
+
+  //  OpaqueExistentialContainer layout.
+  //  struct OpaqueExistentialContainer {
+  //  void *fixedSizeBuffer[3];
+  //  Metadata *type;
+  //  WitnessTable *witnessTables[NUM_WITNESS_TABLES];
+  // };
+  Result<std::pair<Type, uint64_t>>
+  getDynamicTypeAndAddressOpaqueExistential(RemoteAddress object) {
+    uint64_t offset = 3 * sizeof(void *);
```

Comment:
This is a host size calculation.  Please use `TargetOpaqueExistentialContainer`.